components/DepartmentsWalkthrough.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+  <v-container class="Category-Background carousel-view" fluid>
+    <transition-group class="carousel" tag="div">
+      <div
+        v-for="(department, i) in departments"
+        v-bind:key="department.department"
+        v-bind:style="{display: department.display}"
```

Comment:
Not exactly sure what the goal of this is -- why are some dpts not displayed (though they do seem to show up on my local dev)

Reply:
OH I see this is how you carousel. Fine for now! I wonder if there's a suitable open source component out there so we don't have to roll our own though